### DISCUSSION TITLE
Not all songs added to catalog, and error in _get_tags() function.  How do I know which song/directory is the culprit.

### DISCUSSION THREAD

**Discussion by hroussel2016** [ampache/ampache#3742] [Q&A]
Hello.

I just did a fresh installation of Ampache 6.2 on a ubuntu 23.10 with php 8.2.

To test my installation I first added a sample catalog containing two songs.  Everything worked well.  And on the show catalogs page I see a "Last Add" date.

Then I tried a first real catalog (but still small), about 1200 songs.

From the stats I can see it got 1065 songs.  And in the logs is an error which I put below.  It appears to crash somewhere in a get_tags() function.  Is there an easy way to find which song/directory caused the issue?  Thanks.

----------error------------------------------
[Sat Dec 23 21:02:38.719930 2023] [php:error] [pid 14474] [client 127.0.0.1:55698] PHP Fatal error:  Uncaught TypeError: Illegal offset type in /var/www/html/ampache/src/Module/Util/VaInfo.php:580\nStack trace:\n#0 /var/www/html/ampache/src/Module/Util/VaInfo.php(864): Ampache\\Module\\Util\\VaInfo::clean_tag_info()\n#1 /var/www/html/ampache/src/Module/Util/VaInfo.php(359): Ampache\\Module\\Util\\VaInfo->_get_tags()\n#2 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(901): Ampache\\Module\\Util\\VaInfo->gather_tags()\n#3 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(445): Ampache\\Module\\Catalog\\Catalog_local->insert_local_song()\n#4 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(308): Ampache\\Module\\Catalog\\Catalog_local->add_file()\n#5 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(365): Ampache\\Module\\Catalog\\Catalog_local->add_files()\n#6 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(308): Ampache\\Module\\Catalog\\Catalog_local->add_file()\n#7 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(365): Ampache\\Module\\Catalog\\Catalog_local->add_files()\n#8 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(308): Ampache\\Module\\Catalog\\Catalog_local->add_file()\n#9 /var/www/html/ampache/src/Module/Catalog/Catalog_local.php(529): Ampache\\Module\\Catalog\\Catalog_local->add_files()\n#10 /var/www/html/ampache/src/Repository/Model/Catalog.php(4087): Ampache\\Module\\Catalog\\Catalog_local->add_to_catalog()\n#11 /var/www/html/ampache/src/Module/Api/SseApiApplication.php(90): Ampache\\Repository\\Model\\Catalog::process_action()\n#12 /var/www/html/ampache/public/server/sse.server.php(32): Ampache\\Module\\Api\\SseApiApplication->run()\n#13 {main}\n  thrown in /var/www/html/ampache/src/Module/Util/VaInfo.php on line 580, referer: http://localhost/ampache/index.php

**Comment by jamesc2151** [comment]
Try turning on full logging  restart server and try again. 
![image](https://github.com/user-attachments/assets/9e5a5f64-b142-41ef-9959-166cca583106)

**Comment by lachlan-00** [comment]
6.2?

If you need 6 you should at least go up to https://github.com/ampache/ampache/releases/tag/6.6.5

I think your getid3_tag_order in the config file has something that isn't allowed in the list or is empty
